tests/api_tests/test_genomic_api.py

```diff
@@ -522,3 +519,23 @@ def test_ingest_aw5_data_task_api(self, ingest_aw5_mock):
         self.assertEqual(call_json['bucket'], data['bucket_name'])
         self.assertEqual(call_json['job'], GenomicJob.AW5_WGS_MANIFEST)
         self.assertIsNotNone(call_json['file_data'])
+
+    def test_ingest_data_files_task_api(self):
+
+        data = {
+            "file_path": "test_file_path",
+            "bucket_name": "test_bucket",
+        }
+
+        from rdr_service.resource import main as resource_main
+        resource_main.app.testing = True
```

Comment:
I've been trying to find what this might change in the code but haven't been able to. What affect does this have?

Reply:
Tests that API route returns correctly.

Reply:
Sorry, I should have been more specific. I was referring to the line setting `resource_main.app.testing` to `True`.

Reply:
Oh, yea looks like it was unneeded actually and was carried over from other tests, removed refs.
https://github.com/all-of-us/raw-data-repository/pull/2446/commits/c3d0c3504d1f4503bc929b85f0df81b6f23cbb4d